pandas/core/frame.py

```diff
@@ -2326,9 +2326,111 @@ def to_html(
+        """,
+        examples_sub="""
+>>> int_values = [1, 2, 3, 4, 5]
+>>> text_values = ['alpha', 'beta', 'gamma', 'delta', 'epsilon']
+>>> float_values = [0.0, 0.25, 0.5, 0.75, 1.0]
+>>> df = pd.DataFrame({"int_col": int_values, "text_col": text_values,
+...                   "float_col": float_values})
+>>> df
+    int_col text_col  float_col
+0        1    alpha       0.00
+1        2     beta       0.25
+2        3    gamma       0.50
+3        4    delta       0.75
+4        5  epsilon       1.00
+
+Prints information of all columns:
+
+>>> df.info(verbose=True)
+<class 'pandas.core.frame.DataFrame'>
+RangeIndex: 5 entries, 0 to 4
+Data columns (total 3 columns):
+    #   Column     Non-Null Count  Dtype
+---  ------     --------------  -----
+    0   int_col    5 non-null      int64
+    1   text_col   5 non-null      object
+    2   float_col  5 non-null      float64
+dtypes: float64(1), int64(1), object(1)
+memory usage: 248.0+ bytes
+
+Prints a summary of columns count and its dtypes but not per column
+information:
+
+>>> df.info(verbose=False)
+<class 'pandas.core.frame.DataFrame'>
+RangeIndex: 5 entries, 0 to 4
+Columns: 3 entries, int_col to float_col
+dtypes: float64(1), int64(1), object(1)
+memory usage: 248.0+ bytes
+
+Pipe output of DataFrame.info to buffer instead of sys.stdout, get
+buffer content and writes to a text file:
+
+>>> import io
+>>> buffer = io.StringIO()
+>>> df.info(buf=buffer)
+>>> s = buffer.getvalue()
+>>> with open("df_info.txt", "w",
+...           encoding="utf-8") as f:  # doctest: +SKIP
+...     f.write(s)
+260
+
+The `memory_usage` parameter allows deep introspection mode, specially
+useful for big DataFrames and fine-tune memory optimization:
+
+>>> random_strings_array = np.random.choice(['a', 'b', 'c'], 10 ** 6)
+>>> df = pd.DataFrame({
+...     'column_1': np.random.choice(['a', 'b', 'c'], 10 ** 6),
+...     'column_2': np.random.choice(['a', 'b', 'c'], 10 ** 6),
+...     'column_3': np.random.choice(['a', 'b', 'c'], 10 ** 6)
+... })
+>>> df.info()
+<class 'pandas.core.frame.DataFrame'>
+RangeIndex: 1000000 entries, 0 to 999999
+Data columns (total 3 columns):
+    #   Column    Non-Null Count    Dtype
+---  ------    --------------    -----
+    0   column_1  1000000 non-null  object
+    1   column_2  1000000 non-null  object
+    2   column_3  1000000 non-null  object
+dtypes: object(3)
+memory usage: 22.9+ MB
+
+>>> df.info(memory_usage='deep')
+<class 'pandas.core.frame.DataFrame'>
+RangeIndex: 1000000 entries, 0 to 999999
+Data columns (total 3 columns):
+    #   Column    Non-Null Count    Dtype
+---  ------    --------------    -----
+    0   column_1  1000000 non-null  object
+    1   column_2  1000000 non-null  object
+    2   column_3  1000000 non-null  object
+dtypes: object(3)
+memory usage: 188.8 MB""",
+        see_also_sub="""
+DataFrame.describe: Generate descriptive statistics of DataFrame
+    columns.
+DataFrame.memory_usage: Memory usage of DataFrame columns.""",
+    )
```

Comment:
can you indent these

Reply:
or keep the text adjacent to the implementation

Reply:
@jbrockmendel that's a good suggestion - have indented